central/sensor/service/pipeline/complianceoperatorscans/pipeline_test.go

```diff
@@ -141,19 +93,6 @@ func (s *PipelineTestSuite) TestRunReconcileNoOp() {
 	ctx := context.Background()
 
 	s.v1ScanDS.EXPECT().Walk(ctx, gomock.Any()).Return(nil).Times(1)
-	s.v2ScanDS.EXPECT().GetScansByCluster(ctx, fixtureconsts.Cluster1).Return(nil, nil).Times(1)
-
-	err := s.pipeline.Reconcile(ctx, fixtureconsts.Cluster1, reconciliation.NewStoreMap())
-	s.NoError(err)
-}
-
-func (s *PipelineTestSuite) TestRunReconcile() {
```

Comment:
This test was mostly focused on V2 and didn't exist prior to using V2.  Removing it from here but keeping it in the V2 pipeline test.